src/plugins/data/common/search/aggs/buckets/date_histogram.ts

```diff
@@ -116,7 +116,14 @@ export const getDateHistogramBucketAgg = ({
               dateFormat: getConfig('dateFormat'),
               'dateFormat:scaled': getConfig('dateFormat:scaled'),
             });
-            updateTimeBuckets(this, calculateBounds, buckets);
+
+            try {
+              updateTimeBuckets(this, calculateBounds, buckets);
+            } catch (e) {
+              // swallow the error even though the agg is misconfigured
```

Comment:
I think this is safe and shouldn't break anything because this is inside the accessor and previosly threw error only on a first attempt to access. Now at least it would be consistent